packages/react/src/components/NativeSelect/NativeSelect.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { forwardRef } from 'react';
+import type { ForwardedRef, ReactNode, SelectHTMLAttributes } from 'react';
+import cn from 'classnames';
+
+import { InputWrapper } from '../_InputWrapper';
+
+export type NativeSelectProps = {
+  /** Instances of `option` */
+  children?: ReactNode;
+
+  /** Specifies whether the select box is disabled. */
+  disabled?: boolean;
+
+  /** Specifies whether the selected value is valid. */
+  isValid?: boolean;
+
+  /** Label that appears over the select box. */
+  label?: string;
+
+  /** Set to true to enable multiple selection. */
+  multiple?: boolean;
+
+  /** The ID of the `select` element. This will be generated if not provided. */
+  id?: string;
+
+  /** Defines the number of visible options. */
+  size?: number;
+
+  /** The class name for the wrapper element. */
+  wrapperClassName?: string;
+
+  /** The ID of the wrapper element. */
+  wrapperId?: string;
+} & SelectHTMLAttributes<HTMLSelectElement>;
+
+const NativeSelect = forwardRef<HTMLSelectElement, NativeSelectProps>(
+  (
+    {
+      children,
+      disabled = false,
+      id,
+      isValid = true,
+      label,
+      wrapperClassName,
+      wrapperId,
+      ...selectProps
+    },
+    ref: ForwardedRef<HTMLSelectElement>,
+  ) => (
+    <InputWrapper
+      className={wrapperClassName}
+      disabled={disabled}
+      id={wrapperId}
+      inputId={id}
+      isValid={isValid}
+      inputRenderer={({ className, inputId }) => (
+        <select
+          disabled={disabled}
+          id={inputId}
+          ref={ref}
+          {...selectProps}
+          className={cn(className, selectProps.className)}
+        >
+          {children}
+        </select>
+      )}
+      label={label}
+    />
+  ),
+);
+
+NativeSelect.displayName = 'NativeSelect';
```

Comment:
This is not needed as name is infered from const. Suggest we don't override it unless we want to change the name to something else.

Reply:
@Thuneer added this to all components a while ago. I think it was because of issues related to developer tools; for some reason they are not always showing the name of the component if it is not specified like this.

Reply:
When I was trying to convert the React components from JSX to string (to be displayed in code examples for the Storefront), if the displayName property was not present, the component name would look something like this: `<_component_2323<_component_2323>`. I think it was because babel/typescript was doing some magic at the time to make it this way. This is not needed anymore. 

Only time I found the need to use displayName was when I was trying to create compound components 
like `<Header.Left />`.

Then I would do: 
`HeaderLeft.displayName = 'Header.Left';`
`Header.Left = HeaderLeft;`
`export { Header };`

Reply:
I see, I'll remove it then!